spec/models/completed_lesson_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe CompletedLesson, type: :model do
+  context 'associations' do
+    it { is_expected.to belong_to :user }
+    it { is_expected.to belong_to :lesson }
+  end
+
+  describe "validates" do
+    it("should have a default value of completed as true") do
+      # GIVEN 
+      # A new instance of a Completed Lesson without specifying the boolean value
+      user = FactoryBot.create(:user, id:1)
+      lesson = FactoryBot.create(:lesson, id:1)
+      completed_lesson = FactoryBot.create(:completed_lesson, user_id: "1", lesson_id: "1")
```

Comment:
When you set the associations in the factory definitions `factories/models/completes_lessons.rb` you can these three lines with one:
```ruby
completed_lesson = FactoryBot.create(:completed_lesson)
```